frontend/src/app/modules/calendar/wp-calendar/wp-calendar.component.ts

```diff
@@ -92,6 +92,7 @@ export class WorkPackagesCalendarController implements OnInit, OnDestroy {
     jQuery(element).tooltip({
       content: this.contentString(workPackage),
       items: '.fc-content',
+      close: function () { jQuery(".ui-helper-hidden-accessible").remove(); },
```

Comment:
Unrelated fix. tooltips keep spamming aria-live regions in the DOM

Reply:
👍